app/beacon-omop-worker/pyproject.toml

```diff
@@ -26,6 +26,9 @@ psycopg2-binary = "^2.9.9"
 requests="2.31.0"
 python = ">=3.8,<3.11"
 
+[tool.poetry.group.dev.dependencies]
+black = "^24.4.0"
```

Comment:
also take a look at [ruff](https://github.com/astral-sh/ruff) instead of black. it's faster (which helps on pre-commit), and includes isort and more.

Reply:
A bit out of scope for the PR but a good idea for the future